spec/plugins/order_stash_spec.coffee

```diff
@@ -32,10 +32,52 @@ describe 'OrderStash', ->
 
   describe '.constructor', ->
     it 'adds the plugin style to the head', ->
-      expect(window.parent.document.getElementById('sa-order-stash-style')).to.be
+      expect(window.parent.document.getElementById('sa-order-stash-style')).to.not.be.a('null')
```

Comment:
The correct syntax of `be` is to be used with `a` or `an` assertions as chains.([chai documentation](http://chaijs.com/api/bdd/#method_a))

But i prefer more [`exist`](http://chaijs.com/api/bdd/#method_exist) method, so i will change it!